index.html

```diff
@@ -1540,6 +1540,222 @@ <h3>
         </aside>
       </section>
     </section>
+    <section>
+      <h3>
+        `related_applications` member
+      </h3>
+      <p>
+        A <dfn>related application</dfn> is an application accessible to the
+        underlying application platform that has a relationship with the web
+        application.
+      </p>
+      <p>
+        The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+        "manifest">related_applications</dfn></code> member lists <a>related
+        applications</a> and serves as an indication of such a relationship
+        between web application and <a>related applications</a>. This
+        relationship is unidirectional and unless a listed application claims
+        the same relationship, the <a>user agent</a> MUST NOT assume a
+        bi-directional endorsement.
+      </p>
+      <p>
+        Example of usages of the `related_applications` could be a crawler that
+        would use that information to gather more information about the web
+        application or a browser that could suggest a listed application as an
+        alternative if the user wants to install the web application.
+      </p>
+      <p>
+        To <dfn>process the `related_applications` member</dfn>, given
```

Comment:
please reference this in the "Extensions to processing the manifest" section

Reply:
Done!